libknet/bindings/rust/build.rs

```diff
@@ -0,0 +1,30 @@
+extern crate pkg_config;
+
+fn main() {
+    if let Err(e) = pkg_config::probe_library("libknet") {
+	match e {
+	    pkg_config::Error::Failure { .. } => panic! (
+		"Pkg-config failed - usually this is because knet development headers are not installed.\n\n\
+                For Fedora users:\n# dnf install libknet1-devel\n\n\
+                For Debian/Ubuntu users:\n# apt-get install libknet1-dev\n\n\
+                pkg_config details:\n{}",
+		e
+	    ),
+	    _ => panic!("{}", e)
+	}
+    }
+
+    if let Err(e) = pkg_config::probe_library("libnozzle") {
```

Comment:
We can probably drop the libnozzle part here?

Reply:
Thinking about this, we actually need to be linking against the dev tree for the test programs anyway. I need to look into how that's done

Reply:
https://github.com/gyscos/zstd-rs/blob/master/zstd-safe/zstd-sys/build.rs would be an example of having both
- pre-generated bindings commited to git
- compile-time generated bindings
via a feature switch ("bindgen")

as well as
- bundled library source code compiled together with the bindings via cc
- system library discovery + linking via pkg_config
via feature switch ("pkg-config" feature)

Reply:
ok cool